src/deluge/dsp/compressor/rms_feedback.h

```diff
@@ -20,48 +20,56 @@
 #include "definitions_cxx.hpp"
 #include "dsp/filter/ladder_components.h"
 #include "dsp/stereo_sample.h"
+#include "util/fixedpoint.h"
 #include <cmath>
 
 class RMSFeedbackCompressor {
 public:
 	RMSFeedbackCompressor();
-	void setup(q31_t attack, q31_t release, q31_t threshold, q31_t ratio, q31_t sidechain_fc);
 
+	// takes in knob positions in the range 0-ONE_Q31
```

Comment:
Which 2 parameters are the knob positions?

Also, `///` so doxygen picks it up <3

Reply:
All of them, but yeah I'll fix that